qemu/tests/cfg/block_repeat_blockdev_add.cfg

```
@@ -0,0 +1,5 @@
+- block_repeat_blockdev_add:
+    required_qemu = [5.2.0,)
+    type = block_repeat_blockdev_add
+    host_script = block_device/repeat_blockdev_add.sh
```

```suggestion
    host_script = block_device/repeat_blockdev_add_with_qsd.sh
```
I think that we could make a distinction between the regular `repeat_blockdev_add` and it with `qemu-storage-daemon`.

---

updated